woocommerce/woocommerce-android
# [Dynamic Dashboard] Update stats card

Closes: #11316  Closes: #11317


### Description
This PR implements the following changes:
- It updates the Stats card to follow the new design.
- Replace the tabs with a drop-down menu.

The hide button is not implemented yet, I'll add it on a separate PR when we have a final decision following: p1713495594539859-slack-C03L1NF1EA3
And the chart doesn't span to the full width as it's in Figma, I tried playing with the library a bit, and this proved harder than anticipated, so I left it to a separate PR, and it's a nice to have IMO, so we should prioritize the other tasks anyway,

### Testing instructions
1. Make sure the feature flag `DYNAMIC_DASHBOARD` is enabled
2. Open the app, and confirm the `Performance` card is enabled.
3. Confirm the card follows the design.
4. Change date ranges, and confirm it behaves as expected.

### Images/gif
<img width=360 src="https://github.com/woocommerce/woocommerce-android/assets/1657201/ef6b3571-3127-4e51-9fc2-a94a6b21cba7" />
